src/main/java/org/usfirst/frc/team4099/robot/subsystems/Wrist.kt

```diff
@@ -24,12 +35,12 @@ class Wrist private constructor(): Subsystem {
         setWristPower(0.0)
     }
 
-    fun setWristMode(state: WristState) {
+    private fun setWristMode(state: WristState) {
         wristState = state
     }
 
-    fun setWristPower(power: Double) {
-        wristSRX.set(power)
+    private fun setWristPower(power: Double) {
```

Comment:
rename to `setWristPosition`

To clarify, motion magic is basically fancy position control